### PR TITLE
Set proper version

### PR DESCRIPTION
**Pull request by Cruiser13**
Coreshop has wrong version in Pimcore Admin

### CODE REVIEW

File: src/CoreShop/Bundle/CoreBundle/Application/Version.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
     const MAJOR_VERSION = '2';
     const MINOR_VERSION = '0';
     const RELEASE_VERSION = '0';
-    const EXTRA_VERSION = 'alpha.5';
+    const EXTRA_VERSION = '2.0.0-beta.1';
```
Comment (dpfaffenbauer) [inline src/CoreShop/Bundle/CoreBundle/Application/Version.php]: Thats not right, it should be `beta.1`only

**Comment by dpfaffenbauer** [comment]
Oh gosh... Though I already fixed that... Can you make the PR right and then I'll merge it

**Comment by Cruiser13** [comment]
Done

**Comment by dpfaffenbauer** [comment]
thank you

**Pull request merged**